### PR TITLE
Use opencdc in Turbine

### PR DESCRIPTION
**Pull request by lyuboxa**
## Description of change

Fixes <GitHub Issue>

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [ ]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||

### CODE REVIEW

File: pkg/app/config.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ type Config struct {
 	Language ir.Lang           `json:"language"`
 }
 
-// validateAppConfig will check if app.json contains information required
+// validateAppConfig will check if app.json contains information required.
```
Comment (raulb) [inline pkg/app/config.go:20]:
```suggestion
// validateConfig will check if app.json contains information required.
```

File: pkg/server/run.go [hunk 2]
```diff
@@ -5,30 +5,30 @@ import (
...
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
-var _ pb.TurbineServiceServer = (*runService)(nil)
+var _ turbinev2.TurbineServiceServer = (*RunService)(nil)
 
-type runService struct {
-	pb.UnimplementedTurbineServiceServer
+type RunService struct {
```
Comment (raulb) [inline pkg/server/run.go:18]: @lyuboxa Why did we need to export it?

File: .golangci.yml [hunk 3]
```diff
@@ -4,22 +4,17 @@ run:
...
-          - pkg: "errors"
-            desc: "Use github.com/go-errors/errors instead."
-          - pkg: "github.com/pkg/errors"
-            desc: "Use github.com/go-errors/errors instead."
-
-  forbidigo:
-    forbid:
-      - p: ^time.Now$
-        msg: "Use github.com/meroxa/mdpx/core/timeext.Now instead."
+#   depguard:
```
Comment (nassor) [inline .golangci.yml:7]: Is this a complete removal or a way to bypass the check temporarily?

Comment (lyuboxa) [reply .golangci.yml:7]: @nassor temporary, loads of linting errors we need to address.

File: pkg/server/run.go (hunk 2 above)
Comment (lyuboxa) [reply pkg/server/run.go:18]: linter was complaining and rightfully. It is its own type.

**Review by nassor — APPROVED** [review]

**Pull request merged**